sbctl.go

```diff
@@ -80,10 +92,10 @@ func GetESP() string {
 			continue
 		}
 
-		return entryToCheck.Mountpoint
+		return entryToCheck.Mountpoint, nil
 	}
 
-	return ""
+	return "", fmt.Errorf("failed to locate ESP. Is it mounted?")
```

Comment:
`var ErrNoESP = errors.new("missing EFI system partition")` and `return "", ErrNoESP` then we can give guidance in the top level by matching on the error in `cmd/sbctl/main.go`.